README.md

```diff
@@ -60,3 +60,14 @@ add_filter( 'msm_sitemap_entry', 'example_filter_msm_sitemap_entry', 10, 1 );
 ```
 
 Second, if you are filtering the root sitemap, which displays the URLs to the individual sitemaps by date, you will need to filter the `home_url` directly. There is no plugin-specific hook to filter the URLs on the root sitemap.
+
+
+## Filter Sitemap Index
+
+Use the `msm_sitemap_index` filter to exclude daily sitemaps from the index based on date.
+
+```
+add_filter( 'msm_sitemap_index', function( $sitemaps ) {
+	return array_filter( $sitemaps, function( $date ) { return '2017-09-09' < $date; } );
```

Comment:
@lgedeon Does this string comparison actually work?

It appears [to be fetching `post_date`](https://github.com/Automattic/msm-sitemap/blob/a029ddb4c6ad6ecfd0d314003bbc48612a1fb17c/msm-sitemap.php#L667-L669) values of type `datetime`. We probably need to do `strtotime()` and compare with another timestamp here.

Reply:
It does work because the date format used sorts correctly even as a string, but `stringtotime()` is a better solution for production code. My reason for doing it this way here was to keep things simple and easy to read. I can include a larger example if you think that would be helpful.

Reply:
@lgedeon Could you please update the example to use `strtotime()` for all date comparisons?

Reply:
To avoid scope creep, let's just change _this_ instance to `strtotime()` if that's going to be better.

Reply:
@GaryJones Sure! I was referring to both `'2017-09-09'` and `$date` in this particular example. Is that OK?